src/libcore/num/mod.rs

```diff
@@ -1775,9 +1779,9 @@ assert_eq!((-a).div_euc(-b), 2); // -7 >= -4 * 2
 
 
         doc_comment! {
-            concat!("Calculates the remainder `self mod rhs` by Euclidean division.
+            concat!("Calculates the least nonnegative remainder of `self (mod rhs)`.
 
-In particular, the result `n` satisfies `0 <= n < rhs.abs()`.
```

Comment:
I would prefer to keep this inequality in the documentation. This is clearer than "least nonnegative remainder" and "as if by euclidean division algorithm."

Reply:
can do